pytorch_sphinx_theme/static/css/theme.css

```diff
@@ -10715,6 +10715,21 @@ article.pytorch-article table {
   table-layout: fixed;
 }
 
+div.deprecated {
+  color: #b94a48;
+  background-color: #F3E5E5;
+  border-color: #eed3d7;
+  margin-top: 0.5rem;
+  padding: 0.5rem;
+  border-radius: 0.5rem;
```

Comment:
To be consistent with other admonitions, let's make `border-radius: 0` and `border: none`
Also, background color can be more like in the Warning or Deprecated notice in [the issue description](https://github.com/pytorch/pytorch_sphinx_theme/issues/156) or can try #fdede9 as we use it in other parts of the theme.

Reply:
Just to note that currently it’s not an admonition but still keeps a changeset directive (same as original sphinx)